switches/http2/http/HttpDispatcher.cpp

```diff
@@ -256,8 +257,23 @@ namespace Aseba
 	void HttpDispatcher::registerHandler(const Handler& handler, const HttpMethod& method, const strings& uriPath)
 	{
 		handlers[method][uriPath] = handler;
+
+		std::stringstream opId;
```

Comment:
Ok, so we should just add a small comment line explaining that. Also it took me a while to get that `elt` meant `element`.